packages/client-react-streaming/src/DataTransportAbstraction/WrappedApolloClient.tsx

```diff
@@ -79,6 +95,10 @@ class ApolloClientBase extends OrigApolloClient<NormalizedCacheObject> {
     const info = (this.constructor as typeof ApolloClientBase).info;
     this[sourceSymbol] = `${info.pkg}:ApolloClient`;
 
+    for (const warning of warnings) {
+      console.warn(warning, info.pkg, "ApolloClient");
+    }
```

Comment:
This is down here because we need access to `info`, which relies on `this`, which is only available after the `super` call - but we want to remove these options before that call.